Traceback (most recent call last):
  File "/workspaces/SportSync_AI/content_studio/ai_video/generate_final_video.py", line 95, in <module>
    main()
  File "/workspaces/SportSync_AI/content_studio/ai_video/generate_final_video.py", line 43, in main
    c = ImageClip(str(full)).set_duration(seconds).resize(height=1920)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/moviepy/video/fx/resize.py", line 152, in resize
    newclip = clip.fl_image(fl)
              ^^^^^^^^^^^^^^^^^
  File "<decorator-gen-72>", line 2, in fl_image
  File "/home/codespace/.local/lib/python3.12/site-packages/moviepy/decorators.py", line 14, in outplace
    f(newclip, *a, **k)
  File "/home/codespace/.local/lib/python3.12/site-packages/moviepy/video/VideoClip.py", line 936, in fl_image
    arr = image_func(self.get_frame(0))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/moviepy/video/fx/resize.py", line 150, in <lambda>
    fl = lambda pic: resizer(pic.astype('uint8'), newsize)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/moviepy/video/fx/resize.py", line 37, in resizer
    resized_pil = pilim.resize(newsize[::-1], Image.ANTIALIAS)
                                              ^^^^^^^^^^^^^^^
AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'
